pyomo/contrib/mindtpy/util.py

```diff
@@ -106,8 +141,20 @@ def add_feas_slacks(m, config):
 
 
 def var_bound_add(solve_data, config):
-    """This function will add bound for variables in nonlinear constraints if they are not bounded.
-       This is to avoid an unbound master problem in the LP/NLP algorithm.
+    """
+    This function will add bound for variables in nonlinear constraints if they are not bounded. (This is to avoid
+    an unbounded master problem in the LP/NLP algorithm.)
+
+    Parameters
+    ----------
+    solve_data: MindtPy Data Container
+        data container that holds solve-instance data
+    config: ConfigBlock
+        contains the specific configurations for the algorithm
+
+    Returns
+    -------
+    Returns the model with bounds for variables in unbounded nonlinear constraints
```

Comment:
This function does not return anything explicitly.

Reply:
> This function does not return anything explicitly.

I think we can try replacing this function with `fbbt` in single tree implementation.

Reply:
I agree but this is outside of the scope of this PR. Let's include it in our TODO list.